### PR TITLE
test: add capella support to sim multi-fork tests

### LINKED ISSUE #5460: Add capella to the multi-fork sim tests
**Is your feature request related to a problem? Please describe.**

It's an extension to existing sim tests to include the latest hard fork. 

**Describe the solution you'd like**

We already have the **multi-fork** sim tests, which test transitions of multiple forks. Need to add **capella** to that test.

### PR DESCRIPTION
**Pull request by nazarhussain**
**Motivation**

Keep the sim tests up-to-date.

**Description**

Add capella fork support to multi-fork sim tests. 

Closes #5460

**Steps to test or reproduce**

- Run all sim tests

### CODE REVIEW

**Comment by dapplion** [comment]
Looks good overall! To prevent this from happening again, could you implement a check such that:
- look at all forks that are scheduled in mainnet
- require all those forks to be touched by this test

File: packages/cli/test/sim/multi_fork.test.ts [hunk 1]
```diff
@@ -79,10 +81,14 @@ env.tracker.register({
 await env.start({runTimeoutMs});
 await connectAllNodes(env.nodes);
 
-// The `TTD` will be reach around `start of bellatrixForkEpoch + additionalSlotsForMerge` slot
-// We wait for the end of that epoch with half more epoch to make sure merge transition is complete
-await waitForSlot(env.clock.getLastSlotOfEpoch(bellatrixForkEpoch) + activePreset.SLOTS_PER_EPOCH / 2, env.nodes, {
-  silent: true,
+let lastForkEpoch = 0;
+for (const fork of env.forkConfig.forksAscendingEpochOrder) {
```
Comment (dapplion) [inline packages/cli/test/sim/multi_fork.test.ts:87]: Please add a nice comment above this code explaining the purpose of this and motivation

Comment (nazarhussain) [reply packages/cli/test/sim/multi_fork.test.ts:87]: Done

**Review by dapplion — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.9.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.9.0) :tada:

**Pull request merged**